### PR TITLE
v2 tweaks

### LINKED ISSUE #231: v2: unselected options are bold, selected option normal formatted
After migrating to v2, non-highlighted options are bold, focused option is non-bold. Seems unintuitive. Is this expected?

image

### LINKED ISSUE #232: v2: #219 regression: shows "space to select" for survey.Select
Just upgraded to v2 and it looks like what we fixed in #219 has regressed.

I now see the `space to select` message again in `survey.Select` mode.

image

### PR DESCRIPTION
**Pull request by AlecAivazis**
This PR slightly tweaks a few different things:

* Removed bold weight from unhighlighted Select options
* Fixed a panic that would happen in multi selects with no answers
* Removed errant help message in Select

cc @ahmetb 

fixes #231 
fixes #232

### CODE REVIEW

File: select.go [hunk 1]
```diff
@@ -52,10 +51,10 @@ var SelectQuestionTemplate = `
 {{- color "default+hb"}}{{ .Message }}{{ .FilterMessage }}{{color "reset"}}
 {{- if .ShowAnswer}}{{color "cyan"}} {{.Answer}}{{color "reset"}}{{"\n"}}
 {{- else}}
-  {{- "  "}}{{- color "cyan"}}[Use arrows to move, space to select, type to filter{{- if and .Help (not .ShowHelp)}}, {{ .Config.HelpInput }} for more help{{end}}]{{color "reset"}}
+  {{- "  "}}{{- color "cyan"}}[Use arrows to move, type to filter{{- if and .Help (not .ShowHelp)}}, {{ .Config.HelpInput }} for more help{{end}}]{{color "reset"}}
```
Comment (ahmetb) [inline select.go]: this will warrant test updates, no?

**Comment by ahmetb** [comment]
Thanks for quick turnaround. If you don't mind releasing a v2.0.1, I'd happily pick it up, if not I'll pin to a commit once merged.

**Comment by AlecAivazis** [comment]
I'll release it the moment this is merged

**Comment by AlecAivazis** [comment]
this should be available as  `v2.0.1` now

**Comment by ahmetb** [comment]
thanks a lot! appreciate the ultra-fast response.

**Pull request merged**